dpti/hti.py

```diff
@@ -1386,6 +1388,50 @@ def hti_phase_trans_analyze(job, jdata=None):
     return if_phase_trans
 
 
+def run_task(task_dir, machine_file, task_name, no_dp=False):
```

Comment:
It seems like the `run_task` function is duplicated in multiple files. Consider refactoring this into a utility function or a method in a class to avoid code duplication. [important]